packages/react/src/components/Popover/index.test.tsx

```diff
@@ -169,12 +148,8 @@ test('should call onClose on escape keypress', async () => {
 test('should call onClose on clicking outside', async () => {
   const onClose = jest.fn();
   const user = userEvent.setup();
-  render(<Wrapper tooltipProps={{ onClose }} />, {
-    container: mountNode as HTMLElement
-  });
-  const outsideButton = await screen.findByText(/Click Me!/i);
-  expect(outsideButton).toBeTruthy();
-  await user.click(outsideButton);
+  render(<Wrapper tooltipProps={{ onClose }} />);
+  await user.click(document.body);
```

Comment:
We don't need to add any other elements to test clicking outside. We can click on the `body` element instead.